pkg/schema/load/generic.go

```diff
@@ -88,10 +89,38 @@ func buildGenericScuemata(famval cue.Value) (schema.VersionedCueSchema, error) {
 		}
 		major++
 	}
-
+	if lastgvs.actual, err = prepare(lastgvs.actual); err != nil {
+		return nil, err
+	}
 	return first, nil
 }
 
+func prepare(iCUE cue.Value) (cue.Value, error) {
+	iter, err := iCUE.Fields(cue.Definitions(true))
+	if err != nil {
+		return iCUE, err
+	}
+	for iter.Next() {
+		a := iter.Value().Attribute("cuetsy")
+		_, found, err := a.Lookup(0, "targetType")
+		if a.Err() != nil || !found || err != nil {
+			inst, err := rt.Compile("intermedia", `_,@cuetsy(targetType="interface")`)
```

Comment:
```suggestion
			inst, err := rt.Compile("intermedia", `{_,@cuetsy(targetType="interface")}`)
```

Reply:
This is the first piece that needs to be fixed - it needs to be inside a struct.

This:

```cue
{_}, @cuetsy(targetType="interface")}
```

may also work, according to my testing, but i think the correction I made above is what's preferred.

Reply:
i was concerned that we should also add the attribute to the element that is not structure, is that the case?